src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -270,7 +270,37 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
     userEvent.click(screen.getByText(/continue/i));
 
     expect(screen.queryByRole('alertdialog')).to.not.be.ok;
-    expect(screen.baseElement).to.contain.text('canceled');
+    expect(screen.baseElement).to.contain.text(
+      'You canceled this appointment.',
+    );
+  });
+
+  it('should display who canceled the appointment', async () => {
+    const url = '/va/21cdc6741c00ac67b6cbf6b972d084c1';
+    const canceledAppointment = JSON.parse(JSON.stringify(appointment));
```

Comment:
This makes sense for the way this test file is set up, but typically in our tests we call `getVAAppointmentMock()` in the individual test and then update any attributes we need to change. I may make a ticket to bring this file more in line with the rest of our tests.